app/assets/stylesheets/home.scss

```diff
@@ -1,3 +1,7 @@
 // Place all the styles related to the home controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+body {
+  background-image: url("http://robbielane.net/works/mendenhall/photos/MendenhallPan3.jpg");
```

Comment:
URL `http://robbielane.net/works/mendenhall/photos/MendenhallPan3.jpg` should not contain protocol or domain